tt-train/tests/core/n300_utils_test.cpp

```diff
@@ -264,5 +264,5 @@ TEST_F(N300UtilsTest, MorehClipGradNorm) {
 
     ttml::core::MeshToXTensorVariant<float> identity_composer = ttml::core::VectorMeshToXTensor<float>(mesh_shape);
     auto res_back = ttml::core::to_xtensor(tensor, identity_composer)[0];
-    EXPECT_TRUE(xt::allclose(expected_res, res_back, 2e-2F));
+    EXPECT_TRUE(xt::allclose(expected_res, res_back, 2.2e-2F));
```

Comment:
This is because `ttnn::div` used in `ttnn::moreh_clip_grad_norm` now selects the binary_ng implementation. In the legacy implementation, the composite op promoted the tensor to fp32 if it was bf16 and performed the division in that precision instead. Because binary_ng does not perform this floating point promotion, results will be slightly less precise.